### PR TITLE
Sign In HiFi

### PR DESCRIPTION
**Pull request by cindyxzhang**
_Describe your feature here_
- finished HiFi for sign in page according to Kelly's most recent designs
## Related PRs

_Describe any other PRs that might affect/depend on yours_
- don't think anything directly relies on this as this is just frontend styling, no functionality changes
## Testing
- test if functionality for sign in actually works
- links like forget password, activate now, etc. don't work yet as those have not been designed
_List all manual tests done/new unit tests you have implemented_
- try signing in
_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Screenshots
image

_Required for visual changes_

CC: @gabriel-jt @cindyxzhang @hukellyy

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -29,27 +31,68 @@ const SignInScreen: React.FC = () => {
...
+              <h1 className={styles["welcome-title"]}>Welcome Back!</h1>
+              <h4 className={styles["title-subtext"]}>Sign in to continue</h4>
+              <div className={styles["input"]}>
+                <label className={styles["email-input-label"]}>EMAIL</label>
+                <div className={styles["input-field"]}>
+                  <input
+                    id="email"
+                    name="email"
+                    className={styles["email-input-field"]}
+                    {...register("email", { required: true })}
```
Comment (gabriel-jt) [inline pages/index.tsx:54]: Was adding email validation part of this PR or the next one?

File: pages/index.tsx [hunk 2]
```diff
@@ -29,27 +31,68 @@ const SignInScreen: React.FC = () => {
...
+                />
+                <button
+                  className={styles["hide-password-button"]}
+                  onClick={toggleVisibility}
+                >
+                  <Icon type="hidepassword" className={styles.hidepassword} />
+                </button>
+              </div>
+              <div className={styles["forgot-password"]}>
+                {" "}
```
Comment (gabriel-jt) [inline pages/index.tsx:77]: What's the purpose of this space? I removed it on my end and didn't really notice a visual difference

File: pages/index.tsx [hunk 3]
```diff
@@ -29,27 +31,68 @@ const SignInScreen: React.FC = () => {
...
+                <a href="/">Forgot password? </a>
+              </div>
+              <div className={styles["bottom-row"]}>
+                <div>
+                  Haven&apos;t activated your account yet?{" "}
```
Comment (gabriel-jt) [inline pages/index.tsx:82]: nit: Could you use a string here for better readability? 
```suggestion
                 {"Haven't activated your account yet? "}
```

**Review by gabriel-jt — CHANGES_REQUESTED** [review]
Just a few nits/minimal changes!

**Comment by hukellyy** [comment]
Screen Shot 2021-11-08 at 5 21 56 AM
can you make the text field box's height slightly larger here and make sure it's vertically centre aligned with the email/password text?

**Comment by hukellyy** [comment]
Screen Shot 2021-11-08 at 5 23 29 AM
can you vertically centre align this?

File: pages/index.tsx (hunk 1 above)
Comment (cindyxzhang) [reply pages/index.tsx:54]: i will add validation after merging in this one sorry HAHA

**Review by gabriel-jt — APPROVED** [review]
lgtm!

**Pull request merged**